qiskit/transpiler/passes/layout/apply_layout.py

```diff
@@ -75,6 +74,16 @@ def run(self, dag):
             for node in dag.topological_op_nodes():
                 qargs = [q[virtual_phsyical_map[qarg]] for qarg in node.qargs]
                 new_dag.apply_operation_back(node.op, qargs, node.cargs, check=False)
+            if self.property_set["elision_final_layout"]:
+                elission_virtual_map = self.property_set["elision_final_layout"].get_virtual_bits()
+                self.property_set["elision_final_layout"] = Layout(
+                    {
+                        new_dag.qubits[virtual_phsyical_map[bit]]: virtual_phsyical_map[
```

Comment:
lol, $10 if you can figure out who wrote the original code :)

Reply:
Just for the fun of it, the typo seems to have been introduced in #7036 and specifically in commit https://github.com/Qiskit/qiskit/pull/7036/commits/0154230fa4c260d7942b11f39d1ca07456fa612e, which is coauthored by @mtreinish and @kdk. I don't know if it's possible to understand who exactly committed the code, but since Matthew is the one asking the question it has to be Kevin.